winpr/libwinpr/sspi/NTLM/ntlm_av_pairs.c

```diff
@@ -171,14 +172,14 @@ int ntlm_get_target_computer_name(PUNICODE_STRING pName, COMPUTER_NAME_FORMAT ty
 {
 	char* name;
 	int status;
-	DWORD nSize = 0;
-	GetComputerNameExA(type, NULL, &nSize);
-	name = (char*) malloc(nSize);
+	TCHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
```

Comment:
use CHAR instead (since GetComputerNameExA and other ascii function versions are used below)